overlays/kuadrant/kustomization.yml

```diff
@@ -6,6 +6,7 @@ resources:
   - ../../base/mockserver/
   - ../../base/jaeger/
   - ../../base/keycloak-deployment/
+  - ../../base/grafana/
```

Comment:
if `deploy-grafana.sh` would be used, it should not be here (as objects need to be created in sequence with waits.